### PR TITLE
Reactotron Configure

### PR DESCRIPTION
**Pull request by vinod20191992**
Reactotron successfully configured for both android and ios in react native and for connecting reactron to android run adb reverse tcp:9090 tcp:9090 command and refresh android app and for ios just run the app in simulator or device and for any configuration change please see the reactotron-config file

### CODE REVIEW

**Comment by MDHayes** [comment]
You need to update the README. What are we using Reactoron for?

How will you setup with Redux? Redux Saga?

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "eslint": "^6.8.0",
     "jest": "^25.4.0",
     "metro-react-native-babel-preset": "^0.59.0",
-    "prettier": "^1.19.1",
```
Comment (MDHayes) [inline package.json]: Why has the prettier version changed?

Comment (vinod20191992) [reply package.json]: ok let me revert it.

**Comment by MDHayes** [comment]
Cool I'll merge this, please remember to setup Reactotron for Redux etc.

**Pull request merged**